test-requirements.txt

```diff
@@ -1,7 +1,19 @@
-apache-airflow==1.10.11
+apache-airflow==1.10.12
+apache-airflow[gcp]==1.10.12
+apache-airflow[gcp_api]==1.10.12
+apache-airflow[google]==1.10.12
+apache-airflow[postgres]==1.10.12
 codecov==2.1.8
 marquez-python==0.7.5
 pytest==6.0.1
 pytest-cov==2.10.1
 requests==2.24.0
 psycopg2==2.8.6
+mock>=4.0.2
+httplib2>=0.18.1
+google-cloud-bigquery>=1.28.0
+google-auth-httplib2>=0.0.4
+google-api-core>=1.22.2
+google-api-python-client>=1.12.2
+pandas-gbq>=0.13.2
+google-cloud-storage>=1.31.2
```

Comment:
I suspect that we don't actually need the google libraries for the tests

Reply:
CI failed, and i need to add those whatever CI needed.